android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageView.kt

```diff
@@ -0,0 +1,421 @@
+				) {
+					Text(
+						text = stringResource(R.string.ensuite_bathroom),
+						style = SubletrTypography.displaySmall,
+					)
+					Text(
+						/* TODO listing details needs ensuite bathroom info */
+						text = stringResource(R.string.no),
+						color = primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Row(
+					modifier = Modifier
+						.fillMaxWidth(ELEMENT_WIDTH),
+					verticalAlignment = Alignment.CenterVertically,
+					horizontalArrangement = Arrangement.SpaceBetween,
+				) {
+					Text(
+						text = stringResource(R.string.total_rooms_in_house),
+						style = SubletrTypography.displaySmall,
+					)
+					Text(
+						/* TODO listing details needs total rooms in house */
+						text = uiState.listingDetails.rooms.toString(),
+						color = primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Row(
+					modifier = Modifier
+						.fillMaxWidth(ELEMENT_WIDTH),
+					verticalAlignment = Alignment.CenterVertically,
+					horizontalArrangement = Arrangement.SpaceBetween,
+				) {
+					Text(
+						text = stringResource(R.string.total_bathrooms_in_house),
+						style = SubletrTypography.displaySmall,
+					)
+					Text(
+						/* TODO listing details needs total bathrooms in house */
+						text = stringResource(R.string.no),
+						color = primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.s)),
+				)
+
+				Row(
+					modifier = Modifier
+						.fillMaxWidth(ELEMENT_WIDTH),
+					verticalAlignment = Alignment.CenterVertically,
+					horizontalArrangement = Arrangement.SpaceBetween,
+				) {
+					Text(
+						text = stringResource(id = R.string.description),
+						color = primaryTextColor,
+						style = SubletrTypography.displaySmall,
+					)
+				}
+
+				Spacer(
+					modifier = Modifier.height(dimensionResource(id = R.dimen.xs)),
+				)
+
+				TextField(
+					modifier = Modifier
+						.wrapContentSize(align = Alignment.Center)
+						.border(
+							BorderStroke(
+								dimensionResource(id = R.dimen.xxxs),
+								textFieldBackgroundColor
+							),
+							shape = RoundedCornerShape(dimensionResource(id = R.dimen.xs))
+						)
+						.fillMaxWidth(ELEMENT_WIDTH),
+					readOnly = true,
+					shape = RoundedCornerShape(size = dimensionResource(id = R.dimen.xs)),
+					colors = TextFieldDefaults.colors(
+						unfocusedContainerColor = secondaryButtonBackgroundColor,
+						focusedContainerColor = secondaryButtonBackgroundColor,
+						unfocusedIndicatorColor = Color.Transparent,
+						focusedIndicatorColor = Color.Transparent,
+					),
+					value = uiState.listingDetails.description,
+					onValueChange = {},
+				)
+			}
+		}
+	}
+}
+
+private const val ELEMENT_WIDTH = 0.9f
+private const val SUBLET_IMAGE_WIDTH = 0.8f
+private val dateFormat = DateTimeFormatter.ofPattern("MM/dd/yyyy")
```

Comment:
Technically bad practice but we can keep it for now since demo.
Can you just add a TODO to make this localized in the future? (Not all languages use "MM/dd/yyyy")

Reply:
_localization_